### PR TITLE
Removed multisig JS part + decoding part

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/learn/learn-accounts.md [hunk 1]
```diff
@@ -480,171 +480,22 @@ while she waits for either Bob or Charlie also to approve the call. When Bob com
...
-
-Click 'Make Transfer' - ensure that Bob is the 'multisig signatory', and click 'Sign and Submit'.
-
-:::note
+:::info
 
-'Multisig message with the call (for final approval)' is automatically enabled; this means
-that, since the transaction will reach the signature threshold, it will execute the actual
-transaction on the chain after adding the second signature.
+This section has been moved to [this support page](https://support.polkadot.network/support/solutions/articles/65000181826-how-to-create-and-use-a-multisig-account).
```
Comment (profradha) [inline docs/learn/learn-accounts.md]: Provide info on where the user should look for and recommend they try it out on Westend first

```suggestion
Check the "How to create a multisig account" section on [this support page](https://support.polkadot.network/support/solutions/articles/65000181826-how-to-create-and-use-a-multisig-account). We recommend trying out the tutorial on [Westend network](../maintain/maintain-networks.md#westend-test-network) - Polkadot's testnet.
```

File: docs/learn/learn-accounts.md [hunk 2]
```diff
@@ -480,171 +480,22 @@ while she waits for either Bob or Charlie also to approve the call. When Bob com
...
 
-You can use Polkadot-JS Apps to decode call data, as the extension only shows raw call data.
-This will allow you to know what you're signing. 
-
-:::note
+:::info
 
-The decoded call data requires you to understand what the events or calls mean in the context of 
-the transaction.
+This section has been moved to [this support page](https://support.polkadot.network/support/solutions/articles/65000181826-how-to-create-and-use-a-multisig-account).
```
Comment (profradha) [inline docs/learn/learn-accounts.md]: Could you reference the support page section that has this information?

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Add pointers to guide the Wiki reader to appropriate sections in the support page

File: docs/learn/learn-accounts.md (hunk 2 above)
Comment (filippoweb3) [reply docs/learn/learn-accounts.md]: Done, see new commit

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**